test/functorch/test_control_flow.py

```diff
@@ -1951,7 +1969,10 @@ def chain_fct(inp):
         unittest.skip,
         lambda params: (
             params["combine_mode"] == "pointwise"
-            and params["device"] == torch.device("cpu")
```

Comment:
Wouldn't it be easier to say `params["device"] != torch.device("cuda")`

Reply:
Probably, this part was already here but I can make this change also if we prefer this.

Reply:
would that also skip for CUDA?  are we wanting to only skip for ROCm?

Reply:
Yes, that'd also skip for CUDA, which we do not want